AFQ/viz/utils.py

```diff
@@ -68,6 +73,10 @@
     "Right Uncinate": tableau_20[17], "UF_R": tableau_20[17],
     "Left Arcuate": tableau_20[18], "AF_L": tableau_20[18],
     "Right Arcuate": tableau_20[19], "AF_R": tableau_20[19],
+    "Left Posterior Arcuate": tableau_20[6],
+    "Right Posterior Arcuate": tableau_20[7],
+    "Left Vertical Occipital": tableau_extension[0],
+    "Right Vertical Occipital": tableau_extension[1],
```

Comment:
Looks like you're not using all the colors in the extension set. Is that intentional?

Reply:
Yes, I added another color to the extension set if we need it in the future, but for now we only have 22 standard bundles (18 original, without the HCC, plus 4 from pARC/VOF). I am making up the colors for the extension set, trying to make them different from the existing 20 colors, but not perfect (for example, here, the VOF color looks similar to the UNC color).

Reply:
I think it's fine!